doc/nrf/libraries/modem/nrf_modem_lib/nrf_modem_lib_socket_offloading.rst

```diff
@@ -4,16 +4,16 @@ Socket offloading
 #################
 
 Zephyr Socket API offers the :ref:`socket offloading functionality <zephyr:net_socket_offloading>` to redirect or *offload* function calls to BSD socket APIs such as ``socket()`` and ``send()``.
-The integration layer utilizes this functionality to offload the socket API calls to the Modem library and thus eases the task of porting the networking code to the nRF91 Series device by providing a wrapper for Modem library's native socket API such as :c:func:`nrf_socket` and :c:func:`nrf_send`.
+The integration layer utilizes this functionality to offload the socket API calls to the Modem library and thus eases the task of porting the networking code to the nRF9x Series device by providing a wrapper for Modem library's native socket API such as :c:func:`nrf_socket` and :c:func:`nrf_send`.
```

Comment:
This need not be changed as here we are talking about the devices, not the socket or SoCs.